lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -154,6 +160,26 @@ module Constants
           'Google::Apis::LoggingV2beta1::LogEntryOperation'
         ]
       }
+
+      # Used as a map key for entries with no locally unique id in the payload.
+      NO_UNIQUE_ID_PRESENT = 'no_locally_unique_id'
+
+      # Metadata Agent support.
+
+      # Use empty string as request path when locally-unique key of monitored
+      # resource can be implicitly inferred by Metadata Agent.
+      IMPLICIT_MONITORED_RESOURCE_UNIQUE_KEY = ''
+
+      DEFAULT_METADATA_AGENT_URL =
+        'http://local-metadata-agent.stackdriver.com:8000'
+
+      # The label name of locally unique id in the json payload. When a record
+      # has this field in the payload, we will use the value to retrieve
+      # monitored resource from Stackdriver Metadata agent.
+      DEFAULT_UNIQUE_ID_KEY = "#{DEFAULT_PAYLOAD_KEY_PREFIX}/locally_unique_id"
```

Comment:
Why do we call this `DEFAULT_UNIQUE_ID_KEY`? Can this key be overridden? If not, let's call it something like `LOCAL_RESOURCE_ID_KEY` or something...

Reply:
Nope, this can not be overridden. I just changed to `LOCAL_RESOURCE_ID_KEY`.